src/components/Observations/ObsCardStats.js

```diff
@@ -1,74 +1,72 @@
 import React from "react";
 import { useTranslation } from "react-i18next";
 import { useTheme } from "react-native-paper";
 
 type Props = {
   item: Object,
-  type?: string,
   layout?: string
 }
-
-const ObsCardStats = ( { item, type, layout }: Props ): Node => {
+/* eslint-disable react-native/no-inline-styles */
+const ObsCardStats = ( { item, layout }: Props ): Node => {
   const { t } = useTranslation( );
   const theme = useTheme( );
   const qualityGrade = checkCamelAndSnakeCase( item, "qualityGrade" );
 
-  const setIconColor = ( ) => {
+  const getIconColor = ( ) => {
+    if ( layout === "grid" ) {
+      return theme.colors.onSecondary;
+    }
     if ( item.viewed === false ) {
       return theme.colors.error;
-    } if ( layout === "grid" ) {
-      return theme.colors.onPrimary;
     }
     return theme.colors.primary;
   };
 
-  const renderIdRow = ( ) => (
-    <ActivityCount
-      count={item.identifications?.length}
-      color={setIconColor( )}
-      accessibilityLabel={t( "Number-of-identifications" )}
-      testID="ActivityCount.identificationCount"
-    />
-  );
-
-  const renderCommentRow = ( ) => (
-    <ActivityCount
-      count={item.comments?.length}
-      color={setIconColor( )}
-      accessibilityLabel={t( "Number-of-comments" )}
-      testID="ActivityCount.commentCount"
-    />
-  );
-
-  const renderQualityGrade = ( ) => (
-    <QualityGradeStatus qualityGrade={qualityGrade} color={setIconColor( )} />
-  );
-
-  const renderColumn = ( ) => (
-    <View>
-      {renderIdRow( )}
-      {renderCommentRow( )}
-      {renderQualityGrade( )}
-    </View>
-  );
-
-  const renderRow = ( ) => (
-    <View className="flex-row absolute bottom-1 justify-between w-full px-2">
-      <View className="flex-row">
-        {renderIdRow( )}
-        {renderCommentRow( )}
+  const iconColor = getIconColor( );
+  const margin = layout === "list" ? "mb-2.5" : "mr-2.5";
+  const flexDirection = layout === "list" ? "flex-column" : "flex-row";
+  return (
+    <View className={classnames(
+      "flex px-2",
+      flexDirection
+    )}
+    >
+      {
+        /*
+        @TODO add ic_id icon
+        <ActivityCount
+          marginClass={margin}
+          count={item.identifications?.length}
+          color={iconColor}
+          accessibilityLabel={t( "Number-of-identifications" )}
+          testID="ActivityCount.identificationCount"
+        />
+        */
+      }
+      <View className={classnames( margin, "flex flex-row items-center" )}>
+        <Image
+          style={{ height: 15, width: 15, tintColor: iconColor }}
+          source={require( "images/ic_id.png" )}
+        />
+        <Body3 className="ml-1" style={{ color: iconColor }}>{item.identifications.length}</Body3>
       </View>
-      {renderQualityGrade( )}
+      <ActivityCount
+        marginClass={margin}
+        count={item.comments?.length}
+        color={iconColor}
+        accessibilityLabel={t( "Number-of-comments" )}
```

Comment:
Nothing needs to change here right now, but I think we need to change the way accessibility labels for for ActivityCount (I've taken a stab at it in #447). I *think* `accessibilityValue` is mostly for interactive controls and that `accessibilityLabel` should be used for informational elements like this instead, and that `accessibilityLabel` shoudl convey the same thing that's conveyed visually, so something like "12 comments" instead of "Number of comments".